Tools/Scripts/webkitpy/common/net/bugzilla/test_expectation_updater.py

```diff
@@ -79,15 +79,15 @@ def argument_parser():
 
 class TestExpectationUpdater(object):
 
-    def __init__(self, host, bugzilla_id, is_bug_id=True, bot_filter_name=None, attachment_fetcher=None):
+    def __init__(self, port, bugzilla_id, is_bug_id=True, bot_filter_name=None, attachment_fetcher=None):
```

Comment:
This looks like this relies off of #20551?

Reply:
See the comment above. (Yes, it does!)